redaxo/src/core/lib/packages/package.php

```diff
@@ -222,10 +222,12 @@ public function getSupportPage($default = null)
      */
     public function includeFile($__file, array $__context = [])
     {
-        extract($__context, EXTR_SKIP);
+        if ($__context) {
+            extract($__context, EXTR_SKIP);
+        }
 
-        if (file_exists($this->getPath($__file))) {
-            return include $this->getPath($__file);
+        if ($content = @include $this->getPath($__file)) {
```

Comment:
`include` liefert ja den return-Wert, falls die Datei `return` nutzt.
Weiß du, wie sich das verhält, wenn in der Datei `return;` aufgerufen wird? Wird dann `null` (wie bei Funktionen) zurückgegeben, oder `true` wie sonst auch bei `include`, wenn `return` nicht genutzt wird?

Reply:
wird `null` zurückgegeben

Reply:
Aber dann passt das hier doch nicht. dann geht es unten ja noch in den anderen include, obwohl der include hier geklappt hatte, die datei bricht bloß mit einem `return;` ab.

Reply:
Hmm ich glaube man kann nicht unterscheiden ob das included file „false“ returned hat, oder das file nicht gefunden wurd...(zumindest nicht am return wert).

Ggf. Kann man mit error-get-last() arbeiten,  oder man lässt es wie es ist... ?

Reply:
Ich nutze `return` jedenfalls öfters in der `boot.php`, die ja gerade über die Methode hier eingebunden wird.
Beispiele:
https://github.com/FriendsOfREDAXO/developer/blob/master/boot.php#L3-L9
https://github.com/yakamara/ytraduko/blob/master/boot.php#L19

Ich denke, ich würde es dann eher so lassen, wie es ist.